src/Uno.UI.Wasm/ts/Windows/UI/Xaml/Animation/RenderingLoopFloatAnimator.ts

```diff
@@ -0,0 +1,89 @@
+﻿namespace Windows.UI.Xaml.Media.Animation {
+	export class RenderingLoopFloatAnimator {
+		static activeInstances: { [jsHandle: number]: RenderingLoopFloatAnimator} = {};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'activeInstances' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5444000)